recipes/mailio/all/test_package/test_package.cpp

```diff
@@ -8,7 +8,4 @@ int main() {
   msg.add_recipient(mailio::mail_address("mailio library", "mailio@gmail.com"));
   msg.subject("smtps simple message");
   msg.content("Hello, World!");
-
-  mailio::smtps conn("smtp.gmail.com", 587);
-  std::cout << msg.content() << "\n";;
```

Comment:
Great catch! These are the things we definetely don't want in the test package code for CCI recipes, so thanks for removing it :)